### PR TITLE
WA Task Event Completion for Release

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4076

### Change description ###
Task event completion

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
@johnbenjamin-hmcts  thanks for making changes

File: src/shared/components/case-editor/case-event-completion/components/case-event-completion-task-reassigned/case-event-completion-task-reassigned.component.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          if (!this.assignedUserName) {
+            this.assignedUserName = 'another user';
+          }
+        });
+      }
+    });
+  }
+
+  public ngOnDestroy(): void {
+    if (this.subscription) {
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-editor/case-event-completion/components/case-event-completion-task-reassigned/case-event-completion-task-reassigned.component.ts:66]: Minor comment - Think Phil went over some of these conditions and added a check of the closed flag to see if the subscription had been unsubscribed previously - possibly to help with tests. Might be worth doing that here? e.g. && !this.subscription.closed

File: src/shared/components/event-start/event-start.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (connorpgpmcelroy) [inline src/shared/components/event-start/event-start.component.html:1]: Is this file necessary?

**Review by connorpgpmcelroy — APPROVED** [review]
Minor comments - However, nothing that would indicate change needs to be made

**Review by udaydenduluri33 — APPROVED** [review]
